README.md

```diff
@@ -1,13 +1,26 @@
-QuickStart
+Essence
 ====
 
-QuickStart is meant to be exactly that, a Sponge plugin that allows you to quickly start your server with essential commands, events, and other tidbits that you might need. 
-Extremely configurable, only loading up the modules you want (and providing a way for plugins to disable modules that they replace the functionality of), and providing a simple and rich API, 
-QuickStart is an elite plugin for providing simple server tasks.
+[![GitHub license](https://img.shields.io/badge/License-MIT-brightgreen.svg?style=flat-square)](/LICENSE.txt) |
+[![GitHub license](https://img.shields.io/badge/Dependency-JitPack-brightgreen.svg)](https://jitpack.io/#EssencePowered/Essence)
+
+* [Source]
+* [Issues] | [![GitHub issues](https://img.shields.io/github/issues/EssencePowered/Essence.svg?style=flat-square)](http://www.github.com/EssencePowered/Essence/issues/)
+* [Wiki]
+* [Downloads] | [![Github Releases](https://img.shields.io/github/downloads/EssencePowered/Essence/total.svg?style=flat-square)](http://www.github.com/EssencePowered/Essence/releases)
+
+### Status
+* Master: [![Travis branch](https://img.shields.io/travis/EssencePowered/Essence/master.svg?style=flat-square)](https://travis-ci.org/EssencePowered/Essence)
+* Dev: [![Travis branch](https://img.shields.io/travis/EssencePowered/Essence/develop.svg?style=flat-square)](https://travis-ci.org/EssencePowered/Essence)
+
+Essence is meant to be exactly that, a Sponge plugin that allows you to quickly start your server with essential
```

Comment:
This made sense when it said "QuickStart" - but not when Essence. Need to think of a better description...